components/places/sql/create_main_triggers.sql

```diff
@@ -22,3 +22,60 @@ FOR EACH ROW WHEN OLD.guid != NEW.guid
 BEGIN
     SELECT RAISE(FAIL, 'guids are immutable');
 END;
+
+-- Tags
+CREATE TEMP TRIGGER moz_tags_relations_afterinsert_trigger
+AFTER INSERT ON moz_tags_relation
+BEGIN
+  UPDATE moz_tags SET
```

Comment:
A few lines up it looks like we're using 4 space indents for SQL, so it might be good to keep with that...